src/components/RestaurantMenuHeader.js

```diff
@@ -0,0 +1,152 @@
+import {
+  FlatList,
+  HStack,
+  Skeleton,
+  VStack,
+  useColorModeValue,
+} from 'native-base';
+import React, {useEffect, useRef, useState} from 'react';
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+
+const styles = StyleSheet.create({
+  sectionMenu: {
+    display: 'flex',
+    flexDirection: 'row',
+    backgroundColor: '#201E1E',
+  },
+  sectionMenuItem: {
+    borderBottomWidth: 1,
+  },
+  sectionMenuItemText: {
+    paddingHorizontal: 24,
+    paddingVertical: 16,
+  },
+});
+
+const RestaurantMenuHeader = ({
+  sections,
+  sectionRef,
+  activeSection,
+  isLoading,
+}) => {
+  const [active, setActive] = useState(0);
+  const ref = useRef(null);
+  const backgroundColor = useColorModeValue('#fff', '#201E1E');
```

Comment:
I'd suggest move as many colours as possible into a global style/theme to be able to re-use them elsewhere